packages/components/src/internal/components/assay/AssayResultDeleteModal.spec.tsx

```diff
@@ -21,5 +21,7 @@ describe('<AssayResultDeleteModal/>', () => {
 
         // Progress
         expect(wrapper.childAt(1).prop('title')).toBe('Deleting 1 assay result');
+
+        wrapper.unmount();
```

Comment:
got it. removed. we have unmount() calls sprinkled throughout our jest tests. I thought for awhile it was required and would cause issues with downstream tests if not done...maybe that was never the case?

Reply:
I have read (perhaps [here](https://github.com/enzymejs/enzyme/issues/911), perhaps out of date) that calling unmount() is actually best practice for more efficient jest test runs.

Reply:
I'm going to investigate implementing the generic solution outlined in that issue on Monday. I would much prefer it to be something that is automatically handled for us.